src/lgtm/scripts/download/grafana.sh

```
@@ -0,0 +1,45 @@
+	enterprise)
+		ARCHIVE="grafana-enterprise-${VERSION}.linux-${ARCH}.tar.gz"
+		;;
+	*)
+		echo "Unsupported edition: ${EDITION}"
+		exit 1
+		;;
+esac
+
+# Prepare
+TEMP_DIR="$(mktemp -d)"
+trap "rm -rf '$TEMP_DIR'" EXIT
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)